Source/WebCore/platform/graphics/texmap/TextureMapperLayer.cpp

```diff
@@ -543,8 +536,9 @@ void TextureMapperLayer::damageWholeLayerDueToTransformChange(const Transformati
 {
     // When the layer's transform changes, we must not only damage whole layer using new transform,
     // but also using old transform to cover the area not affected by layer anymore.
```

Comment:
```suggestion
    // When the layer's transform changes, we must not only damage the whole layer using the new transform,
    // but also using the old transform, as it could previously have affected an area the new one doesn't.
```